packages/format-message-interpret/index.js

```diff
@@ -138,6 +138,21 @@ function interpretSelect (locale, id, children) {
 
 function interpretSimple (id) {
   return function format (args) {
-    return '' + args[id]
+    return '' + getArg(id, args)
+  }
+}
+
+function getArg (id, args) {
+  if (args[id] !== undefined) return args[id]
```

Comment:
I think that's a good way to go. There are probably arguments to ignore `null` and `undefined` values, but I think the `in` check is simple and reasonable. Let's go with that.